### DISCUSSION TITLE
Replace all my search engine with SearxNG

### DISCUSSION THREAD

**Discussion by fabioteixei** [searxng/searxng#2945] [Q&A] [answered]
Hello

Low level question. I already have SearxNG up and running.

Now i want to replace all my search engines with it on:

 - Desktop Browser (Mainly Edge but Chrome as well)
 - Mobile Browser (Android Chrome)
 - Mobile Search Bar

I did not found the information on how to do it except for the Desktop Browser that's prety straightforward. 
But on the Mobile i need help to se if it's even possible.

Can you please help me?

**Comment by austinhuang0131** [comment, marked as answer]
mobile search bar depends on your OS, in many cases it's not possible

for Android Chromium:

1. go to your favourite instance
2. search something
3. go to Chrome settings => search engines
4. the instance should pop up in the list

note that this assumes the instance has a properly-configured `opensearch.xml`

**Reply by fabioteixei** [reply]
Thanks, that worked well.
